dev-app/src/screens/MerchantSelectScreen.tsx

```diff
@@ -22,7 +23,31 @@ export default function MerchantSelectScreen() {
   const [accounts, setAccounts] = useState<Array<IShortAccount>>([]);
   const [isAddPending, setIsAddPending] = useState<boolean>(false);
   const [newAccountKey, setNewAccountKey] = useState<string>('');
+  const [newStripeAccountID, setNewStripeAccountID] = useState<string>('');
+  const connectAccounts = [
+    { value: 'Default', type: ChargeType.Default },
+    { value: 'Direct', type: ChargeType.DirectCharge },
+    { value: 'Destination', type: ChargeType.DestinationCharges },
+  ];
```

Comment:
Since the Destination Charge is a payment specific setting we don't need to keep track of that here and already have an input field on the CollectCardPaymentScreen. 

For the Direct Charge we can simplify the code a bit I think. We don't need to associate Direct Charge Account with the main account. It should be enough to have just an input field to enter an account id and persist the value of that through different merchants. If the app has a stored Direct Charge Account we display that in the input field, if not (or if a user deletes it) just display the input field with no set value.

Reply:
For the Direct Charge we can simplify the code a bit I think. We don't need to associate Direct Charge Account with the main account. It should be enough to have just an input field to enter an account id and persist the value of that through different merchants. If the app has a stored Direct Charge Account we display that in the input field, if not (or if a user deletes it) just display the input field with no set value.